team/README.md

```diff
@@ -0,0 +1,17 @@
+# The team folder
+
+All members of the organizing team should add themselves to the webpage and
+this folder contains a file for each person describing themselves.
+
+## Adding a new member
+Follow these steps to add yourself:
+* Use the provided 'template.yaml' file and copy the file.
+* Rename the file to your name
+   ```
+    mv template.yaml FirstName_LastName.yaml
+   ```
+* Open the file in your favorite editor and fill out the details, 
```

Comment:
For clarity, specify which file to edit (so people don't start editing the template)
* Open the new file you just created....